src/app/auth.tsx

```diff
@@ -0,0 +1,60 @@
+'use server';
+
+import {User, DataModel, UserType, Database} from 'lib/data';
+import {createSession, deleteSession} from './session';
+import {redirect} from 'next/navigation';
+import {revalidatePath} from 'next/cache';
```

Comment:
since this import is not used, it should be removed

Reply:
you can keep it as email but you should use the normalize ID function I added in my latest PR so that the id is all lowercase and has no spaces. I had issues with some budget ids  because they had spaces